app/build.gradle

```diff
@@ -54,29 +54,42 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.3.1'
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
This is added automatically when new Fragment is created, but we don't really need it.

Reply:
faktycznie, mozna wyrzucic